web/src/components/organisms/Project/Schema/index.tsx

```diff
@@ -12,9 +12,9 @@ import WorkspaceCreationModal from "@reearth-cms/components/molecules/Common/Wor
 import FieldList from "@reearth-cms/components/molecules/Schema/FieldList";
 import FieldCreationModal from "@reearth-cms/components/molecules/Schema/FieldModal/FieldCreationModal";
 import FieldUpdateModal from "@reearth-cms/components/molecules/Schema/FieldModal/FieldUpdateModal";
-import ModelCreationModal from "@reearth-cms/components/molecules/Schema/ModelCreationModal";
 import ModelFieldList from "@reearth-cms/components/molecules/Schema/ModelFieldList";
-import SchemaMenu from "@reearth-cms/components/molecules/Schema/SchemaMenu";
+import ModelsMenu from "@reearth-cms/components/organisms/Project/ModelsMenu";
```

Comment:
Only molecules and atoms should be used inside organisms, right?

Reply:
@nourbalaha , I think molecules and other organisms can be used inside organisms. (I checked re:earth codebase for that).
@KaWaite , please correct me if I'm wrong.

Reply:
Questions like this can be answered by looking at reearth-web's component's README [here](https://github.com/reearth/reearth-web/blob/main/src/components/README.md)

Reply:
@KaWaite , it's not prohibited according to README, also in re:earth codebase there are a ton of organisms inside organisms.

Reply:
@issmail-basel @nourbalaha It doesn't prohibit it, but it is better when at all possible to not have organisms using other organisms. Of course, sometimes it might be better for the code to have organisms inside other organisms (I think very often it'll be models). 
The README states organisms are built up of molecules. Just keep this in your minds while you code okay?